src/test/java/org/broadinstitute/hellbender/engine/AssemblyRegionUnitTest.java

```diff
@@ -122,6 +124,19 @@ private void assertGoodReferenceGetter(final byte[] actualBytes, final SimpleInt
         return tests.subList(2,3).toArray(new Object[][]{});       //HACK!
     }
 
+    @Test(expectedExceptions = IllegalArgumentException.class, expectedExceptionsMessageRegExp = "Contig chrNotInReference not found in reference.*")
```

Comment:
I did not know about `expectedExceptionsMessageRegExp` ! And now that I do, I have mixed feelings about it...throwing a more specific exception type is better than checking the exception message text itself.

Reply:
You're right... changing to UserException.MissingContigInSequenceDictionary.